packages/IMAPClient-Core.package/ICAddressBook.class/instance/addEmail..st

```diff
@@ -1,5 +1,4 @@
 action
 addEmail: aString
 
-	self addIfNotPresent: aString
-	
+	self addAllFirstUnlessAlreadyPresent: {aString} asOrderedCollection
```

Comment:
Why hasn't it been an ordered collection before?

Reply:
We wanted to add the new element to the front of the string and  check if it is already present at the same time. This was only function in OrderedCollection that implements it and it takes an OrderedCollection